tests/test_verify_attached_bugs_cli.py

```diff
@@ -39,25 +39,25 @@ def test_verify_bugs_with_sweep_cli(self):
         flexmock(JIRABugTracker).should_receive("login")
         flexmock(BugzillaBugTracker).should_receive("get_config").and_return({'target_release': ['4.6.z']})
         flexmock(BugzillaBugTracker).should_receive("login")
-        flexmock(sweep_cli).should_receive("get_assembly_bug_ids").and_return(set(), set())
```

Comment:
Refactored tests to only test out top level logic and skip re-testing internal funcs like
`_get_blocking_bugs_for` and `get_attached_bugs`, for which we have separate tests